app/controllers/product_drives_controller.rb

```diff
@@ -66,9 +69,9 @@ def show
 
   def update
     @product_drive = current_organization.product_drives.find(params[:id])
-    if @product_drive.update(product_drive_params)
+    @product_drive.attributes = product_drive_params.merge(tags: tags_from_params)
+    if @product_drive.save
```

Comment:
Is there a reason this pattern is different than in create?

Reply:
Not sure why i did that. Fixed in dbe77d7e522d44881c1a6bbe97d71fd62f007fab to be more consistent with other `update` actions.